### PR TITLE
Convert google image data to json schema

### LINKED ISSUE #95: Google: Convert raw image data according to JSON schema
Implement necessary steps to convert raw Google image data according to the JSON schema.

### PR DESCRIPTION
**Pull request by miyunari**
closes #95

### CODE REVIEW

**Comment by FKolwa** [comment]
Changes looking great as usual. Only thing I'd like to add would be tests for your newly added functions to get the code coverage up again. Basically the same thing we did for AWS!

**Comment by miyunari** [comment]
Thank you :smile:  I am currently working on the tests, that's why I made it a draft pull request for now :+1:

File: src/rhelocator/update_images/gcp.py [hunk 1]
```diff
@@ -57,3 +57,65 @@ def normalize_google_images(image_list: list[Any]) -> list[dict[str, str]]:
...
+    Args:
+        image_name: String containing the image name, such as:
+                    rhel-7-9-sap-v20220719
+
+    Returns:
+        Returns the google image version as string
+                    rhel-7-9
+    """
+    res = image_name.split("-", 3)
+    return res[0] + "-" + res[1] + "-" + res[2]
```
Comment (FKolwa) [inline src/rhelocator/update_images/gcp.py:74]: Hmm I'm afraid Google images don't always follow the RHEL-MAJOR-MINOR-vVERSION formula. 
Sometimes the minor is missing: rhel-9-v20220719, rhel-9-arm64-v20220719 etc. If you want to we can wipe up a quick regex for that.

**Review by FKolwa — DISMISSED** [review]

**Comment by FKolwa** [comment]
Just a minor issue with the version parsing that I didn't see before, other than that it looks great!

File: src/rhelocator/update_images/gcp.py (hunk 1 above)
Comment (FKolwa) [reply src/rhelocator/update_images/gcp.py:74]: Just as an example: https://regex101.com/r/CiABs5/1

Comment (miyunari) [reply src/rhelocator/update_images/gcp.py:74]: Oh yeah right - thank you! :)

File: tests/update_images/test_gcp.py [hunk 2]
```diff
@@ -57,3 +60,59 @@ def test_normalize_google_images() -> None:
...
+
+    """Test parsing a google image name with a basic image."""
+    image_name = "rhel-sap-v20220719"
+    version = gcp.parse_image_version_from_name(image_name)
+
+    assert version == "unknown"
+
+
+def test_format_image():
+    """Test transforming a single google image into a schmea approved
```
Comment (major) [inline tests/update_images/test_gcp.py:92]: Typo on `schema` here, but it's minor. :stuck_out_tongue_winking_eye:

**Review by major — APPROVED** [review]

**Pull request merged**